src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -121,6 +121,23 @@ class Reliability(enum.Enum):
     Medium = 'medium'
     Low = 'low'
 
+    @property
+    def invert(self):
+        '''
+        Verbalize the opposite of `value` of reliability to be used in coherent
+        sentences.
+        '''
+        inversions = {
+            'high': 'low',
+            'medium': 'medium',
+            'low': 'high'
+        }
+
+        if (self.value in inversions):
+            return inversions[self.value]
+
+        return 'unknown'
```

Comment:
That's not "nice" Python: no parenthesis around conditions, and there is a simpler way in one line to replace the if + return.

```suggestion
        return inversions.get(self.value, 'unknown')
```